src/pages/Flow/Automations-Tile.vue

```diff
@@ -0,0 +1,112 @@
+<script>
+import CardTitle from '@/components/Card-Title'
+import AutoCard from '@/pages/Dashboard/Automations/AutomationCard'
+
+export default {
+  components: {
+    CardTitle,
+    AutoCard
+  },
+  props: {
+    flow: {
+      required: true,
+      type: Object
+    }
+  },
+  data() {
+    return {
+      loadingHook: 0,
+      loadCards: false,
+      sortedHooks: []
+    }
+  },
+  watch: {
+    hooks() {
+      this.sortHooks()
+    }
+  },
+  methods: {
+    async sortHooks() {
+      this.loadCards = true
+      const hooks = this.hooks ? [...this.hooks] : []
+      let filtered = hooks.filter(
+        hook => hook.event_tags.flow_group_id === this.flow.flow_group_id
+      )
+      const slaFlows = hooks.filter(
+        hook => hook.event_tags.flow_sla_config_id != null
+      )
+      for (let i = 0; i < slaFlows.length; i++) {
+        const { data } = await this.$apollo.query({
+          query: require('@/graphql/Automations/flow-config-by-pk.gql'),
+          variables: {
+            flowSLAConfigId: slaFlows[i].event_tags.flow_sla_config_id[0]
+          }
+        })
+        if (
+          data.flow_sla_config_by_pk.flow_groups[0].flow_group_id ==
+          this.flow.flow_group_id
+        ) {
+          filtered.push(slaFlows[i])
+        }
+      }
+      const sorted = filtered.sort(
+        (a, b) => new Date(b.created) - new Date(a.created)
+      )
+      this.sortedHooks = sorted
+      this.loadCards = false
+    }
+  },
+  apollo: {
+    hooks: {
+      query() {
+        return require('@/graphql/Automations/hooks.gql')
+      },
+      variables() {
+        return {}
+      },
+      loadingKey: 'loadingHook',
+      update: data => data?.hook
+    }
+  }
+}
+</script>
+
+<template>
+  <v-card class="pa-2 mt-2" tile>
+    <CardTitle title="Automations" icon="fa-random" />
+
+    <v-card-text class="full-height position-relative">
+      <v-row>
+        <v-col cols="12" class="pb-0">
+          <div
+            >Displaying automations for this flow. To see all automations or
```

Comment:
Should we display a message here when there are no automations or will that be too redundant?

Reply:
Agreed. We should have a 'No flows associated with this hook' message.